pdns/pdns_recursor.cc

```diff
@@ -1132,6 +1134,91 @@ int followCNAMERecords(vector<DNSRecord>& ret, const QType& qtype)
   return rcode;
 }
 
+int getFakeAAAARecords(const DNSName& qname, ComboAddress prefix, vector<DNSRecord>& ret)
```

Comment:
Suggest to use a 'const &' for prefix.

Reply:
Note that it won't save the copy since we modify it later in the function (I did not change that code, I guess we could).

Reply:
Oh, missed that. That code sure looks ugly.